openpype/modules/deadline/plugins/publish/submit_publish_job.py

```diff
@@ -275,7 +275,7 @@ def _submit_deadline_post_job(self, instance, job, instances):
         args = [
             "--headless",
             'publish',
-            rootless_metadata_path,
+            '"{}"'.format(rootless_metadata_path),
```

Comment:
I think this will break if the space is not there. Rather use `subprocess.list2cmd(args)` when filling `"Arguments"`.

Reply:
Not sure about that. Apparently even Python isn't really sure if it should be used at all - https://discuss.python.org/t/why-is-subprocess-list2cmdline-not-documented/25044/11

Reply:
TL;DR 
`subprocess.list2cmdline()` is not multiplatfrom compatible, doesn't work with `shell=True`, might be deprecated in future, etc.

Reply:
> subprocess.list2cmdline() is not multiplatfrom compatible

It is not, but it handles spaces the same way for all of them, you don't want to do multiplatform strange things.

> doesn't work with shell=True

From what I see in the forum, it's misunderstanding of function usage and purpose.

> might be deprecated in future, etc.

I don't think so. None of subprocess commands would work.

If you don't want to use it, then at least make sure the metadata path is stripped where is used, because arguments handler can pass `"c:/path_without_space/file"` as string `"c:/path_without_space/file"` instead of `c:/path with space/file`.

Reply:
As far as I know, this is used directly in Deadline API and who knows how it works there :) I suppose it needs to be tested. BTW notes like this in CPython subprocess code:

```python
# NOTE: We intentionally exclude list2cmdline as it is
# considered an internal implementation detail.  issue10838.
```